src/components/AgGrid/EntityFilter.jsx

```diff
@@ -1,15 +1,15 @@
 import { useGridFilter } from 'ag-grid-react';
 import React, { useCallback, useEffect, useState } from 'react';
-import {useSelector} from "react-redux";
-
+import { useSelector } from 'react-redux';
+import Select from 'react-select';
```

Comment:
Is react-select already in the repo, or do you need that in the package.json ?  If it's new, I'll defer to Valerio and Paul about whether we need that, or should use something else (I have no idea)

Reply:
It is already in package.json.

Reply:
"react-select": "^5.8.0"

Reply:
Ah, cool, thanks @sweng66 .  I didn't see it when I pulled main.

Reply:
https://github.com/alliance-genome/agr_literature_ui/blob/main/package.json

Reply:
You are right. For some reason, it is in my local version. I added the following to package.json.

"react-select": "^5.8.0"

Good catch! Thanks. @azurebrd